### PR TITLE
Fix for IE

### PR DESCRIPTION
**Pull request by maximuk**
#### WHAT DOES THIS PR DO:
* added fix for IE (was used wrong syntax)
#### WHERE SHOULD THE REVIEWER START:
* Diff

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const sizes = require('./sizes');
-const variables = require('./variables');
+var sizes = require('./sizes');
+var variables = require('./variables');
```
Comment (rbardini) [inline index.js]: Why change `const` to `var`? `const` is supported by IE11.

Comment (maximuk) [reply index.js]: I know. But it's bad practice to use `const` in node_modules

Comment (froskie) [reply index.js]: A bad practice is to enable the users to access the file directly. You author your code the way you want and you EXPORT/BUILD them for consume properly transpiled for general use.

Comment (maximuk) [reply index.js]: @froskie should we create an issue and fix it in future?

**Pull request merged**